terraform/modules/database/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_db_instance" "api3-operations" {
+  identifier             = "api3-operations"
+  instance_class         = "db.t3.micro"
+  allocated_storage      = 5
+  engine                 = "postgres"
+  engine_version         = "13.1"
```

Comment:
It looks like [14 is currently supported](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.FeatureSupport.Extensions).